app/components/personal_details/view.rb

```diff
@@ -4,25 +4,26 @@ module PersonalDetails
   class View < GovukComponent::Base
     include SanitizeHelper
 
-    def initialize(data_model:, has_errors: false, editable: false)
+    def initialize(data_model:, has_errors: false, editable: false, is_minimal: false)
```

Comment:
Struggled with naming this boolean keyword argument. Other suggestions are welcome. I didn't want it coupled to the idea of lead schools.

Reply:
I think `minimal` is probably fine. It's inconsistent with `editable` if we make it `is_` though so just `minimal`.